app/code/core/Mage/Index/Model/Observer.php

```diff
@@ -20,6 +20,9 @@
 
 class Mage_Index_Model_Observer
 {
+
```

Comment:
You could remove the empty line after class start.

Reply:
I didn't want to 'berserk' through the whole file to make the PR more concise. But it would be a pleasure to update all comments and so on, I will take this as a hint that this is indeed wanted and start right now ;-)

Reply:
You introduced the empty line though in this special case. Or at least Github displays it as one of your additions 🤔.

Reply:
Updated.